### DISCUSSION TITLE
Channel is displayed twice

### DISCUSSION THREAD

**Discussion by thisar93** [nextgenhealthcare/connect#5678] [Q&A]
After deploying a channel, it is displayed twice in the dashboard.
Unfortunately I can't upload a screenshot because I always get an error...

If I select the channel (no matter which entry in the dashboard) and undeploy, then both go away again.

Has anyone had the same experience? Is there a solution for this?

**Comment by jonbartels** [comment]
I've seen it before too. I never found a root cause, I'm guessing a cache.

**Comment by kirbykn2** [comment]
I've seen this before.  What version are you using?

**Reply by thisar93** [reply]
We use Mirth 4.2.0.

**Reply by pacmano1** [reply]
Does closing and re-opening the admin tool fix it?   I have seen it also, but I think re-opening the admin tool fixes it.

**Reply by thisar93** [reply]
After a reboot, that doesn't happen anymore. Funny... :)

Thanks
